content/concepts/consensus-network/federated-sidechains.md

```diff
@@ -0,0 +1,91 @@
+
+Federators, similar to validators on the mainchain but only live on the sidechain, jointly control the door accounts using the built-in multi-signature scheme available on the XRP Ledger. The door account has a signers list which consists of signing keys of all federators on the sidechain. 
+
+When a door account receives a transaction, the federator signs the transaction and broadcasts this information to other federators on the sidechain.
+Simultaneously, federators also listen for signed transactions from other federators and collect them.
+
+A transaction is valid only when 80% of the signers on the list have signed. A quorum is reached when 80% of federators have signed and the transaction is submitted to the door account and an asset is issued to the destination.
+
+When an asset is issued from the mainchain to the sidechain, that asset is locked up in the door account. The asset is only unlocked on the mainchain when a transaction from the sidechain to the main chain is submitted. Transactions within the sidechain are not visible to the servers on the sidechain. 
+
+
+## Terminology
+
+_Sidechain_: A sidechain is an independent ledger which provides a way to transfer assets from the XRP Ledger to the sidechain, and return assets from the sidechain back to the XRP Ledger. Each sidechain has its own consensus algorithm and transaction types and rules. 
+ 
+_Federator_: A server on the sidechain that listens for triggering transactions on both the main chain and the side chain. Each federator has a signing key associated with it that is used to sign transactions. A transaction must be signed by a quorum of federators before it can be submitted. Federators are responsible for creating and signing valid response transactions, collecting signatures from other federators, and submitting transactions to the main chain and side chain.
+
+_Main chain_: XRP Ledger where assets originate and where assets will be locked while being used on the side chain. For most applications, the main chain will be the XRP Ledger Devnet, Testnet, or Mainnet.
+
+_Side chain_: Custom independent ledger where proxy assets for the locked main chain assets are issued. Side chains may have rules, transactors, and validators that are very different from the main chain. Proxy assets on the side chain can be sent back to the main chain where they will be unlocked from the control of the federators.
+
+_Door account_: An account controlled by the federators. There are two door accounts: one on the main chain and one on the side chain. Cross chain transactions are started by users sending assets to a door account. Main chain to side chain transactions cause the balance to increase on the main chain door account and the balance to decrease on the side chain door account. It is called a "door" because it is the mechanism to move assets from one chain to another - much like going between rooms in a house requires stepping through a door.
```

Comment:
```suggestion
_Door account_: An account controlled by the federators. There are two door accounts: one on the mainchain and one on the sidechain. Cross chain transactions are started by users sending assets to a door account. Mainchain to sidechain transactions cause the balance to increase on the mainchain door account and the balance to decrease on the sidechain door account. It is called a "door" because it is the mechanism to move assets from one chain to another—much like going between rooms in a house requires stepping through a door.
```

- "mainchain" and "sidechain" as one word, for consistency
- use a real emdash